ui/pages/send/send-footer/send-footer.stories.js

```diff
@@ -1,30 +1,37 @@
 import React from 'react';
-import { action } from '@storybook/addon-actions';
-import { boolean } from '@storybook/addon-knobs';
 
 import SendFooter from './send-footer.component';
 
 export default {
   title: 'Pages/Send/SendFooter',
   id: __filename,
+  argTypes: {
+    clearSend: { action: 'Cancel Button Pressed' },
+    sign: { action: 'Next Button Pressed' },
+    from: { control: 'object' },
+    disabled: { control: 'boolean' },
+    mostRecentOverviewPage: { control: 'text' },
+    sendErrors: { control: 'object' },
+  },
 };
 
-export const DefaultStory = () => {
-  const disabled = boolean('Disabled', false);
+export const DefaultStory = (args) => {
   return (
     <SendFooter
-      clearSend={() => action('Cancel Button Pressed')()}
-      sign={() => action('Next Button Pressed')()}
-      // The other props below are only to make the component show no error
-      from={{ address: '' }}
+      {...args}
       history={{ push: () => undefined }}
       addToAddressBookIfNew={() => undefined}
-      disabled={disabled}
-      mostRecentOverviewPage=""
       resetSendState={() => undefined}
-      sendErrors={{}}
     />
   );
 };
```

Comment:
This can be simplified now that we have moved the function type props to action controls

```
export const DefaultStory = (args) => {
  return <SendFooter {...args} />;
};
```

Reply:
done